### DISCUSSION TITLE
Access to discord bot without a listening for messages

### DISCUSSION THREAD

**Discussion by mucsacso03** [Rapptz/discord.py#9598] [Q&A]
Hi,
I would like to send messages to a discord server and get already sent messages, but I do not receive any input from the server. If I start this python discord bot it stays in a loop and blocks my application. How can I access the connection in the bot to send simple queries (POST, GET) when needed?

Thank you!

**Comment by IAmTomahawkx** [comment]
It would be helpful to see your code, however it should be noted that you do not need to call run()/start() if your only goal is to send messages (and not receive anything from discord). What you'll need to do instead is call client.login and then after that you can make calls to send those messages (using partial objects to get the channel object you need)

**Reply by mucsacso03** [reply]
Thank you for the information. I would like to do the thing you mentioned: ` What you'll need to do instead is call client.login and then after that you can make calls to send those messages (using partial objects to get the channel object you need)`
What do you mean by partial objects?
